### PR TITLE
第二周作业-张浩

### PR DESCRIPTION
**Pull request by CCheungh**
import torch
import torch.nn as nn
import numpy as np

ce_loss = nn.CrossEntropyLoss()  #先定义用交叉熵作为损失函数

pred = torch.FloatTensor([[0.3, 0.1, 0.3, 0.8],  # 写一个浮点张量作为预测值
                          [0.9, 0.2, 0.9, 0.3],
                          [0.5, 0.4, 0.2, 0.7],
                          [0.5, 0.4, 0.4, 0.5]])

target = torch.LongTensor([1,2,0,3])  # 定义目标值
loss = ce_loss(pred, target)  #计算预测值和目标值的Loss

print(loss,"torch输出交叉熵")

### CODE REVIEW

**Comment by GSYBD** [comment]
删掉不需要提交的文件，不要影响其他人目录下的文件。重新拉取在提交
